### PR TITLE
Content updates to beta.get.gov (domains/choosing + domains/moving)

### PR DESCRIPTION
**Pull request by gabydisarli**
These updates to domains/choosing + domains/moving partially satisfy the ticket [#1282](https://github.com/cisagov/manage.get.gov/issues/1282)

this update does NOT include updating the check availability section on domains/choosing. That work is being tracked in ticket [#1219](https://github.com/cisagov/manage.get.gov/issues/1219). Because the check availability section is not being updated here, as an interim solution, I've linked to the home page where we have the feature. 

## Ticket

Partially satisfies [#1282](https://github.com/cisagov/manage.get.gov/issues/1282)

## Changes

- Content updates to domain/choosing
- Content updates to domain/moving

## Setup

if reviewing on Github, just go to the files I've updated and review content changes there. These should follow the google doc content for: 

1. [domains/choosing](https://docs.google.com/document/d/1n_TAWrivkDh0RwF56gGbLJ1xoNWuH7Ww1Jt4mZC0XS8/edit#heading=h.30j0zll)
2. [domains/moving](https://docs.google.com/document/d/1EL6pBxmLGMj3_b726H2Ai4yY3N7ukufW1Vu3cdTwE8M/edit?usp=sharing)

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

**Comment by Katherine-Osos** [comment]
@gabydisarli The "check availability" links are still highlighted in yellow and aren't linking to the homepage. ([view page in sandbox](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/gd/update-beta.get.gov/domains/choosing/#))

**Comment by Katherine-Osos** [comment]
@gabydisarli Looks like the "check availability" links aren't updated on the ["moving" page](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/gd/update-beta.get.gov/domains/moving/#find-out-if-you're-eligible-for-financial-assistance) either. 

I also made a few [comments in the Google doc](https://docs.google.com/document/d/1EL6pBxmLGMj3_b726H2Ai4yY3N7ukufW1Vu3cdTwE8M/edit) to point out some other links that don't work.

**Review by h-m-f-t — APPROVED** [review]

**Comment by Katherine-Osos** [comment]
@gabydisarli  Let me know when these pages are ready to review again!

**Comment by gabydisarli** [comment]
> @gabydisarli Let me know when these pages are ready to review again!

@Katherine-Osos ready to go!

File: pages/domains/domains_choosing.md [hunk 1]
```diff
@@ -104,42 +103,41 @@ Examples:
 ### Counties
 This organization type includes counties, parishes, or boroughs.
 
-Most county .gov domains must include the two-letter state abbreviation or the full state name. County names that aren’t shared by any other city, county, parish, town, borough, village or equivalent in the U.S., at the time a domain is granted, can be requested without referring to the state. Counties can include “county” in their domain to distinguish it from other places with similar names. We use the [Census Bureau’s National Places Gazetteer Files](https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.html){.usa-link--external} to determine if county names are unique.
+Most county .gov domains must include the two-letter state abbreviation or the full state name. County names that aren’t shared by any other city, county, parish, town, borough, village or equivalent in the U.S., (at the time a domain is granted) don’t have to refer to their state in their domain name. Counties can include “county” in their domain to distinguish it from other places with similar names.
```
Comment (h-m-f-t) [inline pages/domains/domains_choosing.md:106]: > equivalent in the U.S., (at the time a domain is granted)

Can we move the comma to the end of the parentheses?

File: pages/domains/domains_moving.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Read more about each of these steps below.{.checklist}
 - [Plan to move your email to .gov](#plan-to-move-your-email-to-.gov)
 - [Get familiar with domain security best practices](#get-familiar-with-domain-security-best-practices)
 - [Develop a communications plan](#develop-a-communications-plan)
-- [Find out if you're eligible for financial assistance](#find-out-if-you're-eligible-for-financial-assistance)
+- [Find out if you’re eligible for financial assistance](#find-out-if-you‘re-eligible-for-financial-assistance)
```
Comment (h-m-f-t) [inline pages/domains/domains_moving.md:33]: Unfortunately, the move to `’` from `'` means the ID becomes `#find-out-if-you%E2%80%99re-eligible-for-financial-assistance` and this link doesn't work anymore. 

😢 @gabydisarli

Comment (michelle-rago) [reply pages/domains/domains_moving.md]: Fixed

Comment (h-m-f-t) [reply pages/domains/domains_moving.md]: Nevermind! @michelle-rago fixed this at [b91fda3](https://github.com/cisagov/getgov-home/pull/168/commits/b91fda3df24a2a6ed2b8ed0cb5c16f2c51dcb80e).

File: pages/domains/domains_choosing.md (hunk 1 above)
Comment (michelle-rago) [reply pages/domains/domains_choosing.md]: Moved 🚗

**Review by michelle-rago — APPROVED** [review]
Looks good, Gaby!

File: pages/domains/domains_moving.md (hunk 2 above)
Comment (gabydisarli) [reply pages/domains/domains_moving.md]: @therealslimhsiehdy should I change these back to ' to make this work? I know we talked a lot about this...

File: pages/domains/domains_choosing.md (hunk 1 above)
Comment (gabydisarli) [reply pages/domains/domains_choosing.md]: @michelle-rago @h-m-f-t I'm actually realizing that the comma should have never been there to begin with. It's not in the google doc version of this page, or in the request form, where this same content appears.

**Review by Katherine-Osos — APPROVED** [review]

**Pull request merged**